### DISCUSSION TITLE
Login link to a specific page

### DISCUSSION THREAD

**Discussion by sroussey** [supabase/supabase#2132] [Questions] [answered]
Right now, a magic link always points to the home page of a domain (no path). I have setup public pages to not check for login to make them as fast as possible. The logged in area, as well as the signin page, checks to see if logged in. 

Right now, you can signup at https://embarc.com/ after clicking the SignIn button, but when you get the link you land on the home page again and must click the signup button again.

Under _Supabase / Authentication / Settings / Site URL_  I put in **https://embarc.com/signin** but it only takes the base of the URL.

Suggestions?

**Comment by inian** [comment, marked as answer]
Can you set the `Additional Redirect URLs` @sroussey? I think that respects the full path, the path must be an exact match though

**Reply by sroussey** [reply]
I did that to no effect. :/

**Reply by inian** [reply]
Hmm..this might be related to this issue then https://github.com/supabase/gotrue/issues/123

**Reply by sroussey** [reply]
Not really, unless there is an undocumented way when using supabase.auth.login to pass another url to use, otherwise, how would it know?

**Reply by emschwartz** [reply]
@sroussey You should set the Site URL to `https://embarc.com` (just the base URL) and then use the `redirectTo` parameter in the [`signIn`](https://supabase.io/docs/reference/javascript/auth-signin) method set where the user should be redirected after login (for example, `redirectTo: 'https://embarc.com/logged/in/area'`).

Note that setting additional paths or query parameters in the `redirectTo` parameter will only work for the Site URL (**not** the Additional URLs) until https://github.com/supabase/gotrue/issues/99 is implemented.

**Reply by sroussey** [reply]
I did not see that in the docs. Yikes. Trying now, thanks. The additional domains will be helpful for beta sites, etc. Thank you!

**Reply by J0** [reply]
Hi for any future readers of this thread do note that we support [wildcard patterns for Additional Redirect URIs](https://supabase.com/blog/beta-update-may-2022) as of May 2022
